atsha204a/tests/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+project(s96-204_tests C)
+
+cmake_minimum_required(VERSION 3.0.2)
+
+find_package(OpenSSL REQUIRED)
+include_directories(${OPENSSL_INCLUDE_DIR})
+
+file(GLOB SRC ${CMAKE_SOURCE_DIR}/src/*.c)
+list(REMOVE_ITEM SRC ${CMAKE_SOURCE_DIR}/src/main.c)
+
+add_executable(${PROJECT_NAME} EXCLUDE_FROM_ALL ${SRC} tests.c)
```

Comment:
Not a big things, but similar to how you remove `main.c` above, you could also use something like `INSERT` to add tests.c to `${SRC}`. By doing so you have all "include/exclude" together and then you could just in the end here use `${SRC}` and nothing else.

Reply:
You're right, it's inconsistent. How about the proposed change? The rationale is that moving towards a library, main should not be under there in the first place, and will be eventuall moved to a different subdirectory with its own CMakeLists.txt. The top level makefile will then build the library, and we can have separate targets for tests and main and other examples the make use of the library.

Reply:
Just in case github does not show what has been updated between the original and the updated commit: It's atsha204a/CMakeLists.txt and atsha204a/tests/CMakeLists.txt

Reply:
Alright, this section still looks the same, but I can see that you've got rid of the GLOB. This is fine, let's deal with other tweaks later on. Want me to merge this now?

Reply:
Yes please. These parts are to be updated soon anyway.